lua/modules/configs/tool/search.lua

```diff
@@ -0,0 +1,129 @@
+return function()
+	local builtin = require("telescope.builtin")
+	local extensions = require("telescope").extensions
+	require("search").setup({
```

Comment:
```suggestion
	require("modules.utils").load_plugin("search", {
```

I think our custom loader should come in handy here now 😂

Reply:
wow, sorry i don't know i can just sign off suggestion changes!

Reply:
> wow, sorry i don't know i can just sign off suggestion changes!

nvm I didn't know that either until I saw a repo's maintainer submit my suggestions in less than a second (\*hyperbole). I was like 🤯 🤣